GlotPress/GlotPress-WP
# Move the highlight label from the active projects to the inactive

## Problem

As is explained in https://github.com/GlotPress/GlotPress/issues/1335, GlotPress shows a small label:
- In the breadcrumb of each active project.
- In the subproject list, at the right of each active subproject.

![image](https://github.com/GlotPress/GlotPress/assets/1667814/a66a2eff-72d3-4f9e-9ede-901f592e7805)

GlotPress doesn't show this label when the user access to the translation set, selecting a translation language. 

![image](https://github.com/GlotPress/GlotPress/assets/1667814/b4e38249-bef6-4bcb-9fb9-e55ceb86744b)

Fixes https://github.com/GlotPress/GlotPress/issues/1335.


## Solution

This PR inverts the bubble appareance, showing them only: 

- In the subproject list, in each inactive project. The active projects don't have the bubble.

![image](https://github.com/GlotPress/GlotPress/assets/1667814/8546a4d5-b974-4fb9-bc4d-b0dfa77028bc)

- In the breadcrumb of each inactive project, without or with the translation set (language) selected.

![image](https://github.com/GlotPress/GlotPress/assets/1667814/34baf240-e0a6-4c96-9a40-d9f4c501e725)

![image](https://github.com/GlotPress/GlotPress/assets/1667814/3f4e823c-043e-48ba-9b14-b5ecaf287736)

The active projects don't have the bubble in the breadcrumb.

![image](https://github.com/GlotPress/GlotPress/assets/1667814/94e9d0c7-a5b5-4137-a78f-31fa36591103)

![image](https://github.com/GlotPress/GlotPress/assets/1667814/17878889-022f-4ba0-887e-6c191d9204bd)

I have decided to use the "**Inactive**" word instead of "**Archived**", because the project status is **active** or **inactive**, not **archived**. 

![image](https://github.com/GlotPress/GlotPress/assets/1667814/8bb153af-f7e5-4de5-bcb6-205c2ff20288)



## Testing Instructions

To test this change, you only need to navigate between a project with active and inactive subprojects, to see the new approach for the bubbles.
